jep/0000/README.adoc

```diff
@@ -0,0 +1,315 @@
+== Specification
+
+All Jenkins core components use GitHub issues for tracking bugs and feature requests.
+
+This covers every repository where the team 'Core' is the maintainer:
+
+- link:https://cs.github.com/?scopeName=All+repos&scope=&q=repo%3Ajenkins-infra%2Frepository-permissions-updater+%40core[GitHub code search in repository-permissions-updater] (requires access to beta search)
+- link:https://github.com/orgs/jenkinsci/teams/core/repositories[Core team repositories in GitHub] (requires jenkinsci GitHub org membership)
+
+=== Migration
+
+Migration steps are tracked in the link:https://docs.google.com/document/d/1urTCIXbcgZ06zBZHD4l9os3MhHTrIXXTunW7WDbYPpk/edit?usp=sharing[runbook].
+
+=== Process
+
+1. Newly opened issues are labelled with `needs-triage`.
+2. Support requests are redirected to the community forum via automation or saved replies.
+3. Bugs are reproduced and prioritized based on the impact.
+4. If an issue needs more information the label `triage/needs-information` is added and a comment explaining what is needed is added.
+5. After an issue has been triaged the `triage/accepted` label is added and a `sig/?` label is added if there's an appropriate sig to direct it to.
+6. SIG's are automatically notified of newly labelled issues by an link:https://github.com/jenkins-infra/helpdesk/blob/main/.github/workflows/autonotifier.yaml[autonotifier].
+7. SIG's are encouraged to run regular triage sessions to keep on top of the issues.
+
+=== LTS backporting
+
+See link:https://www.jenkins.io/download/lts/#backporting-process[Backporting Process] for the Jira process.
+
+If a pull request requires backporting a pull request will be created cherry-picking the change into the `stable-x` line.
+This will be automatically labelled as `into-lts`.
+
+It will be reviewed against the standard LTS criteria and either accepted and merged or rejected and closed.
+
+Pull requests that need to be backported but are not ready yet, should have the label `lts-candidate` applied to them.
+There will be reviewed during the LTS backporting window and the release lead will backport any remaining labelled pull requests.
+
+After a backport pull request is created the `lts-candidate` label will be removed, unless it was rejected but still eligible in a future backport.
+
+=== Documentation
+
+Contributing guides in Jenkins core and jenkins.io document that Jenkins core components use GitHub issues.
+
+== Motivation
+
+Jira is too disconnected from the code of Jenkins and the other core components.
+
+Historically most large open source projects used tools like bugzilla and Jira. Now there are many that use GitHub:
+- https://github.com/apple/swift
+- https://github.com/kubernetes
+- https://github.com/golang/go
+- https://github.com/rust-lang/rust
+
+The swift organization migrated in https://forums.swift.org/t/swift-bugs-are-moving-to-github-issues-and-we-need-your-help/56125
+
+The key reasons to migrate away are:
+
+=== Developer and user experience
```

Comment:
Worth to mention that you don't need to sign up on accounts.jenkins any longer in order to submit an issue or feature request and can use your regular GH account.